fireblocks_defi_sdk_py/web3_bridge.py

```diff
@@ -77,11 +77,16 @@ def send_transaction(self, transaction: dict, note="") -> dict:
                 address = self.external_wallet_address
             destination = DestinationTransferPeerPath(ONE_TIME_ADDRESS,
                                                       one_time_address={"address": address})
+        amount = transaction.get("value")
```

Comment:
maybe 

```
wei_amount = transaction.get("value", "0") 
amount = str(Web3.fromWei(wei_amount, 'ether'))
```

Reply:
str() cast is redundant.

Reply:
some cast is needed (str or float) because fromWei returns Decimal object and therefore raises
"TypeError: Object of type Decimal is not JSON serializable"

Reply:
[Docs says](https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#fromwei) it will always return a string.

Reply:
It is an [incompatibility](https://web3py.readthedocs.io/en/stable/web3.main.html#web3.Web3.fromWei) between the JS and python libraries